csi/server/plugin/opensds/node_test.go

```diff
@@ -52,7 +52,7 @@ func TestNodeGetCapabilities(t *testing.T) {
 		&csi.NodeServiceCapability{
 			Type: &csi.NodeServiceCapability_Rpc{
 				Rpc: &csi.NodeServiceCapability_RPC{
-					Type: csi.NodeServiceCapability_RPC_UNKNOWN,
+					Type: csi.NodeServiceCapability_RPC_STAGE_UNSTAGE_VOLUME,
```

Comment:
Need to add unit tests to do stage/unstage volume, not just report capabilities.

Also there are no tests for node publish/unpublish.  Need to add tests for that too but that can be a separate PR.

Reply:
Currently these can only be tested manually, but not unit tests.